src/thunderbots/software/test/ai/world/robot_capabilities.cpp

```diff
@@ -0,0 +1,42 @@
+#include "ai/world/robot_capabilities.h"
```

Comment:
Can we move this to src/thunderbots/software/ai/world/robot_capabilities_test.cpp? We've changed the folder structure so we don't have a test folder anymore

Reply:
fixed